src/angularComponent/ngcOmniboxFieldDirective.js

```diff
@@ -30,7 +30,7 @@ export default function ngcOmniboxFieldDirective($document) {
           '{{::omnibox.autofocus === \'true\' || undefined}}');
       input.setAttribute('ng-disabled', 'omnibox.ngDisabled()');
 
-      tElement.html(input.outerHTML);
+      tElement[0].appendChild(input);
```

Comment:
Although wait, if I'm setting `.html('')` anyway, why not just use my original method of doing `.html(input.outerHTML)`? What's the advantage now?